projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -626,6 +626,28 @@ export interface FeatureTogglesInterface {
    *       in the future together with this feature toggle.
    */
   allPageMetaResolversEnabledInCsr?: boolean;
+
+  /**
+   * When enabled, allows to provide extended formats and media queries for <picture> element if used in MediaComponent.
+   *
+   * For proper work requires `pictureElementFormats`  provided in media config:
+   *  ```ts
+   * provideConfig({
+   *   pictureElementFormats: {
+   *    mediaQueries: {
+   *     'max-width': '767px',
+   *      ...
+   *    },
+   *    width: 50,
+   *    height: 50,
+   *   },
+   * })
+   * ```
+   *
+   * After activating this toggle, new inputs in `MediaComponent` — specifically
+   * `width`, `height`, and `size` — will be passed to the template as HTML attributes.
+   */
+  useExtendedMediaComponent?: boolean;
```

Comment:
`nitpick:` What do you think about `useExtendedMediaComponentConfig` or `useExtendedMediaComponentConfiguration` - now as I look at the current name, it suggests what you mentioned offline - that there is some `MediaComponentV2`

Reply:
Changed